welcome.php

```diff
@@ -62,6 +67,8 @@ public function __construct()
         }
 
         if (Module::isInstalled($this->name)) {
+            $this->displayHeader = $this->smarty->getTemplateVars('display_header');
```

Comment:
Two questions:
- shouldn't this be fetched dynamically in the hook at the moment it's called? instead of each Module instanciation (even, maybe, in cases where the smarty vars are not set yet)
- is it the only way to get this variable from the PHP side? I would have thought the variables would be accessible directly in the template? it's more out of curiosity, I understand that for `hookDisplayBackOfficeHeader` it's better to check it beforehand to avoid adding useless css and js files

Reply:
Without the JS part, the module will display infos you cannot interact with. All the purpose of this module (if I understand it well) is to onboard the user into PS usage